infrastructures/infrastructure-gce/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/GCEInfrastructure.java

```diff
@@ -108,7 +108,8 @@
     protected String connectorIaasURL = "http://" + generateDefaultRMHostname() + ":8080/connector-iaas";
```

Comment:
We can add the suffix of the 'connectorIaasURL' to the properties file also.

Reply:
Done for both the suffix and the prefix.